core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/propagation/AbstractPropagationTaskExecutor.java

```diff
@@ -383,6 +383,20 @@ protected Uid delete(
 
             connector.delete(objectClass, uid, null, propagationAttempted);
             result = uid;
+            // SYNCOPE-1809 remove uidOnCreate attribute, if any
```

Comment:
are such comments really needed? We've got `git blame`

Reply:
No, going to remove.